collect_app/src/main/java/org/odk/collect/android/utilities/ToastUtils.java

```diff
@@ -0,0 +1,42 @@
+package org.odk.collect.android.utilities;
+
+import android.content.res.Resources;
+import android.widget.Toast;
+
+import org.odk.collect.android.application.Collect;
+
+
+public class ToastUtils {
+
+    private static final int LENGTH_SHORT = Toast.LENGTH_SHORT;
```

Comment:
Please remove these constants. You should just refer directly to `Toast.LENGTH_xx`. There is no value to saving their values locally.